apps/backend/prompts/github/pr_followup_orchestrator.md

```
@@ -45,13 +45,77 @@ Note: GitHub's API tells us IF there are conflicts but not WHICH files. The find
 
 ## Available Specialist Agents
 
-You have access to these specialist agents via the Task tool:
+You have access to these specialist agents via the Task tool.
+
+**You MUST use the Task tool with the exact `subagent_type` names listed below.** Do NOT use `general-purpose` or any other built-in agent - always use our custom specialists.
+
+### Exact Agent Names (use these in subagent_type)
+
+| Agent | subagent_type value |
+|-------|---------------------|
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)